src/PasswordInputStateful/index.jsx

```diff
@@ -0,0 +1,254 @@
+/*
+ * Copyright (c) 2018 dunnhumby Germany GmbH.
+ * All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the LICENSE file
+ * in the root directory of this source tree.
+ *
+ */
+
+import React                from 'react';
+import PropTypes            from 'prop-types';
+
+import { PasswordInput }    from '../index';
+
+export default class PasswordInputStateful extends React.Component
+{
+    static propTypes =
+    {
+        /**
+         *  ARIA properties
+         */
+        aria : PropTypes.objectOf( PropTypes.oneOfType( [
+            PropTypes.bool,
+            PropTypes.number,
+            PropTypes.string,
+        ] ) ),
+        /**
+         *  Extra CSS class name
+         */
+        className             : PropTypes.string,
+        /**
+         *  CSS class map
+         */
+        cssMap                : PropTypes.objectOf( PropTypes.string ),
+        /**
+         *  Tooltip message text (string or JSX)
+         */
+        errorMessage          : PropTypes.node,
+        /**
+         *  Error Tooltip is displayed
+         */
+        errorMessageIsVisible : PropTypes.bool,
+        /**
+        *   Error message position relative to the icon
+        */
+        errorMessagePosition  : PropTypes.oneOf( [ 'top', 'topLeft' ] ),
```

Comment:
`inputContainer` component has been removed, so there is no need for error message related props.

Reply:
Fixed 🔨